bindings/go/go.mod

```diff
@@ -0,0 +1,7 @@
+module opendal.apache.org/go
+
+go 1.20
```

Comment:
Go 1.19 is still supported

Reply:
This pull request will not be merged in its current form. Please prioritize other more important issues. Thank you.